RELEASE_NOTES.md

```diff
@@ -1,6 +1,34 @@
 # Release Notes
 
 <!-- newest release at the top please) -->
+# 2019/10/02 Prod Release Notes
+
+## Ingest
+### Broker v0.10.1
+* Fix linking
+### Validator v0.6.8
+* Fix retrying of file validation
+* Security Fixes
+### UI v0.11.1
+* Remove unnecessary logging
+* Added link to project view from the submission view
+
```

Comment:
I think these are from an older version of the release notes. We released other modules to production yesterday.